src/main/java/org/resitrack/entity/House.java

```diff
@@ -0,0 +1,9 @@
+package org.resitrack.entity;
+
+import java.util.List;
+
+public class House {
+    private List<Person> members;
+    private int numberHouse;
+    private String householdHead;
```

Comment:
I think `householdId `is a better name for variable holding a household id value. What do you think?
```suggestion
    private String householdId;
```